logs_http.install

```diff
@@ -1,16 +1,35 @@
 <?php
 
+use Drupal\Core\Logger\RfcLogLevel;
+
 /**
  * @file
  * Install, update and uninstall functions for the Logs HTTP module.
  */
 
 /**
+ * Implements hook_install().
+ *
+ * Set all logs_http settings variables.
+ */
+function logs_http_install() {
+  $config = \Drupal::configFactory()->getEditable('logs_http.settings');
+  $config->set('enabled', TRUE);
```

Comment:
Looking at core examples, I believe you can only use the numeric value (3). You can add a comment next to it to indicate it's `RfcLogLevel::ERROR`